RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java

```diff
@@ -116,10 +116,8 @@ public void holdPosition() {
  	
     public double getPosition() {
     	// Returns the position of the elevator
-    	
-    	double position = potentiometer.get();
     	System.out.println("We got the current position of the elevator.");
```

Comment:
This line might want to be either removed or updated to include the potentiometer's current position.